src/App.js

```diff
@@ -64,10 +74,11 @@ class App extends Component {
         <Header />
         <Search updateCurrentDisplay={this.updateCurrentDisplay}/>
         <ConcertDisplay concertData={this.state.concertData}
-                        setlistData={this.state.setlistData}
-                        currentSearch={this.state.currentSearch}/>
+                        setlistData={this.state.currentSetlist}
```

Comment:
Could probably go back together one day and make our prop naming conventions a little more consistent too...